gitlab-build-pack/Jenkinsfile

```diff
@@ -0,0 +1,113 @@
+#!groovy.
+
+node  {
+ //echo sh(returnStdout: true, script: 'env')
+ 
+ def gitlab_credentialsId 	= env.REPO_CREDENTIAL_ID
+ def gitlab_repo_base	    = env.REPO_BASE
+ def gitlab_repo_url        = "http://" + gitlab_repo_base + "/" + env.REPO_LOC + "/"
+
+ def jenkins_username  = env.JENKINS_USERNAME
+ def jenkins_password  = env.JENKINS_PASSWORD
+
+ def gitlab_repo_name  = env.gitlabSourceRepoName
+ def gitlab_branch     = env.gitlabBranch
+ def service_type = ''
+ def service_name = ''
+ def domain_name  = ''
+ 
+ echo "gitlab_credentialsId:gitlab_credentialsId"               
+ echo "gitlab_repo_url:$gitlab_repo_url"	
+ echo "gitlab_repo_name:$gitlab_repo_name"
+ echo "gitlab_branch:$gitlab_branch"
+ 
+ try{
+        sh 'rm -rf $gitlab_repo_name*'
+    	dir(gitlab_repo_name)
+    	{
+    		checkout([$class: 'GitSCM', branches: [[name: '*/master']], doGenerateSubmoduleConfigurations: false, extensions: [], submoduleCfg: [], userRemoteConfigs: [[credentialsId: gitlab_credentialsId, url: gitlab_repo_url + gitlab_repo_name + '.git']]])
+    	   
+    	   //Find the service type	
+    	   if (fileExists('build.api')) {
+				service_type = "api"
+    	   } else if(fileExists('build.lambda')) {
+				service_type = "lambda"
+    	   } else if(fileExists('build.website')) {
+				service_type = "website"
+    	   }
+
+            //Read the JenkinsFile to find the service name and domain
```

Comment:
This approach is not correct. We can't read a JenkinsFile and look for things to understand the service name and domain. 

Use deployment_env instead.

Reply:
Changed the approach by looking into deployment-dev.yml